package.json

```diff
@@ -19,15 +19,22 @@
     "node": ">=10"
   },
   "dependencies": {
-    "pull-stream": "^3.6.0"
+    "debug": "^4.3.2",
+    "pull-pushable": "^2.2.0",
+    "pull-stream": "^3.6.0",
+    "ssb-db2": "^2.5.2",
+    "ssb-ebt": "^8.0.0",
+    "ssb-meta-feeds": "~0.26.0",
```

Comment:
Might be good to bump this to latest 0.28. Also ssb-index-feed-writer seems to be using older version. And ssb-fixtures an even older version (0.21).

Reply:
I wonder if this might be a good time to make 0.28 = version 1? Seems pretty stable

Reply:
@arj03 I don't know of anyone using the APIs of metafeeds in production, and I still won't use them in Manyverse, so I'm fine to delay labeling with version 1.